contrib/NeonVM/neonvm.c

```diff
@@ -600,8 +678,13 @@ void exec_CALLP(struct tagTExecutor *self)
     const uint8_t *func = self->object->strtable[val];
 
     if (strcmp((const char*)func, "print") == 0) {
-        const char *s = pop(self->stack);
+        const char *s = top(self->stack)->string; pop(self->stack);

```

Comment:
If your pop() frees memory, then you're about to print memory pointed to by 's' that has already been freed.